src/api/controllers/articlesController.js

```diff
@@ -0,0 +1,66 @@
+import model from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import errorMessage from '../../helpers/constants/error.messages';
+import errorSender from '../../helpers/error.sender';
+/**
+ * @class
+ */
+export default class Articles {
+  /**
+   * This function gets all the articles that are stored in the article table
+   *
+   * @static
+   * @param {*} req the request
+   * @param {*} res the response to be sent
+   * @memberof Articles
+   * @returns {Object} res
+   */
+  static async getArticles(req, res) {
+    const { Article } = model;
+    const { page } = req.query;
+    const articles = await Article.findAndCountAll();
+    const limit = 10;
+    let offset = 0;
+    const pages = Math.ceil(articles.count / limit);
+    offset = limit * (page - 1);
+    const currentPage = Math.floor(offset / limit) + 1;
+
+    const pagedArticles = await Article.findAll({
+      limit,
+      offset
+    });
+    res.status(status.OK).json({
+      Page: pages,
```

Comment:
@truestbyheart make these keys: `pages`, `count`, and `articles`, instead of `Page`, `Count ` etc

Reply:
```pages``is to show the number of pages you will have according to your limit and ```Count``` is to show the number of articles you have 

````
 "pages": 1,
    "Count": 2,
    "currentPage": 1,
    "Articles":[]```

* pages -shows the number of pages you will get for the limit you set.
* count - is the number of articles
* currentPage - shows the page your in 
* articles - are the articles returned from the database